### PR TITLE
feat: introduce the Chain selector component

### PR DESCRIPTION
**Pull request by juanmahidalgo**
### closed
image

### opened

image

### closed

image

### opened (and switching to polygon in this example)

image

### CODE REVIEW

File: src/components/ChainSelector/ChainSelector.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      <Modal
+        className="dui-chain-selector__modal"
+        open={showModal}
+        onClose={() => setShowModal(false)}
+      >
+        <ModalNavigation title={title} />
+        <ModalContent>
+          <ul className="dui-chain-selector__list">
+            {chains.map((chain) => {
+              console.log('chain: ', chain)
```
Comment (cazala) [inline src/components/ChainSelector/ChainSelector.tsx:62]: oops console log

**Review by cazala — CHANGES_REQUESTED** [review]
On mobile we might want to make the button smaller, or perhaps only show the chain icon instead of icon + name, otherwise it overlaps with the DCL logo:

Screenshot 2024-02-07 at 12 44 21

**Comment by juanmahidalgo** [comment]
> On mobile we might want to make the button smaller, or perhaps only show the chain icon instead of icon + name, otherwise it overlaps with the DCL logo:
> 
> Screenshot 2024-02-07 at 12 44 21

Done!

image

image

**Review by cazala — APPROVED** [review]
🍏!

**Review by cazala — APPROVED** [review]

**Pull request merged**